app/styles/main.scss

```diff
@@ -1028,31 +1028,4 @@ html, body {
   height:100%;
 }
 
-/* Video Button */
```

Comment:
Was this deleted on purpose, the video button seems to be a different issue from organizations.